db/statistics.go

```diff
@@ -155,24 +155,27 @@ func WriteValidatorStatisticsForDay(day uint64) error {
 		}
 
 		logrus.Info(start, end)
-		numArgs := 3
+		numArgs := 4
```

Comment:
**Minor:** This value never changes and should be placed above the for-loop, right? Applies to various locations.
Placing it above the for-loop would enable you to calculate the batch-size dynamically too :)

Reply:
moved and made dynamic